### DISCUSSION TITLE
album covers

### DISCUSSION THREAD

**Discussion by clstaudt** [feugy/melodie#29] [Ideas]
Melodie looks great, and promising (it's high time for an Apple Music alternative). Can we have melodie parse and show album art from the Apple Music library folder please?

**Comment by feugy** [comment]
Hello Christian.

I don't have a Mac, so I don't really know where this folder usually is. Can you tell me where it is please?
Right now, Melodie is automatically searching for [local covers](https://github.com/feugy/melodie/blob/ec3e92ae8382757885ae9bb8f0e8c42a206b337a/common/core/lib/providers/local/cover-finder.js#L10-L27) in track's parent folder. Then it'll search on Discogs & AudioDB if you configure them.

I could definitely add a list of "common" folders, with sensible defaults, and make it configurable.

**Reply by feugy** [reply]
I would also need to know how iTunes is naming their artwork.
It's easy to relate cover/folder files, since they live besides the music files. In the event of a folder grouping all artwork, I'll need to know how to match a given artwork and the music files.

**Reply by clstaudt** [reply]
Unfortunately I don't know much about the internals of Apple Music. However, I know that Apple Music is able to use album art embedded into the mp3 file, while Melodie is not. 

In general, for me Melodie displays no artwork at all.

**Reply by feugy** [reply]
Ah!
This is a completely different thing 😄 

Starting from v2.0.0-alpha.1, Mélodie automatically extracts the cover embedded in music files so it can be used as an album cover.
This should solve this issue!

You'll have to:
- install the latest [beta](https://github.com/feugy/melodie/releases/tag/v2.0.0-beta.1) version
- remove your watched folders in settings
- restart Mélodie twice, so it contains nothing (there will be a reset button some day, sorry for that)
- add your watch folders in settings, and hopefully enjoy your covers!

Let me know if it works for you.
